src/FixedDataTableBufferedRows.js

```diff
@@ -78,27 +81,32 @@ class FixedDataTableBufferedRows extends React.Component {
     var rowClassNameGetter = props.rowClassNameGetter || emptyFunction;
     var rowsToRender = this.props.rowsToRender || [];
 
-    this._staticRowArray.length = rowsToRender.length;
+    if (props.isScrolling) {
+      // we are scrolling, so don't display any rows which lie outside the viewport
```

Comment:
Maybe also mention it's still important to render them so we don't cause mounts & unmounts

Reply:
Sure will do